change/@fluentui-foundation-legacy-2020-10-20-18-13-15-hooks-utilities-rename.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
check change failed. you need to update the change type for `-beta` packages to be `prerelease`